dist/pom.xml

```diff
@@ -77,7 +77,6 @@
         <jdk11.buildvers>
             312
             321
-            331
```

Comment:
I think instead of just removing 331, we should file an issue to make 331 compile with JDK11, then comment 331 out with the link to the issue

Reply:
331 should compile after #6962